packages/fluentui/docs/src/examples/components/Skeleton/Usage/SkeletonExampleList.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import { Skeleton, Image, List } from '@fluentui/react-northstar';
+import { useBooleanKnob } from '@fluentui/docs-components';
+import * as _ from 'lodash';
+
+const SkeletonExampleList = () => {
+  const [loading] = useBooleanKnob({
+    name: 'Loading',
+    initialValue: true,
+  });
+
+  return (
+    <List>
+      {_.times(5, index => (
+        <List.Item
+          key={index}
+          styles={{
+            backgroundColor: index % 2 === 0 ? '#f5f2f2' : 'transparet',
```

Comment:
```suggestion
            backgroundColor: index % 2 === 0 ? '#f5f2f2' : 'transparent',
```

?

---

Is it intentional at all?